salesagility/SuiteCRM
# Fixed #7402 - Popup selects are broken

## Description



Fixes an issue where list-view popups select wouldn't pass back records.

## Motivation and Context

Issue reference: #7402 

## How To Test This

1. Accounts -> contact subpanel -> select.
2. Select multiple contacts and check that the data is passed back correctly.
3. Test as many popups and callback functions as possible.


## Types of changes


### Final checklist
